cve_bin_tool/cli.py

```diff
@@ -273,6 +274,18 @@ def main(argv=None):
         help="comma-separated list of checkers to enable",
     )
 
+    exploit_checker_group = parser.add_argument_group("Exploits")
+    exploit_checker_group.add_argument(
+        "--exploits",
+        action="store_true",
+        help="check for exploits from found cves",
+    )
+    exploit_checker_group.add_argument(
+        "--update_exploits",
```

Comment:
```suggestion
        "--update-exploits",
```
we have been following a kebab case structure for these tags